core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageAutoFeatureStep.java

```diff
@@ -470,7 +620,8 @@ public void addReflectiveField(Map<String, ReflectionInfo> reflectiveClasses, Re
         String cl = fieldInfo.getDeclaringClass();
         ReflectionInfo existing = reflectiveClasses.get(cl);
         if (existing == null) {
-            reflectiveClasses.put(cl, existing = new ReflectionInfo(false, false, false, false, false));
+            reflectiveClasses.put(cl, existing = new ReflectionInfo(false, false, false, false, false,
+                    cl.equals("io.quarkus.it.corestuff.SomeSerializationObject")));
```

Comment:
This is a test file right? Why is it in the core?

Reply:
This seems like a something which shouldn't be there. Thanks for catching this. I'll fix it.

Reply:
fixed, thanks again

Reply:
Looks like the test file is still referenced here

Reply:
Sorry for not fixing it correctly. I can not imagine why it is still there, because I remember removing it. I'll fix it again and double check it.

Reply:
@gastaldi I's fixed, I double checked it. There is no occurrence of `SomeSerializationObject` in  `NativeImageAutoFeatureStep`. Thanks a lot for validating it.